extensions/amp-analytics/0.1/cookie-writer.js

```diff
@@ -149,10 +150,14 @@ export class CookieWriter {
       return BASE_CID_MAX_AGE_MILLIS;
     }
 
-    // convert cookieMaxAge (sec) to milliseconds
-    return (
-      Number(inputConfig['cookieMaxAge']) * 1000 || BASE_CID_MAX_AGE_MILLIS
-    );
+    const cookieMaxAgeNumber = Number(inputConfig['cookieMaxAge']);
+    if (!cookieMaxAgeNumber) {
```

Comment:
nit: Few corner cases: 
cookeMaxAgeNumber = 123.12345, which 123123.45 couldn't convert to time. 
cookeMaxAgeNumber = 0
cookeMaxAgeNumber < 0

Could we add test cases accordingly. Thank you.

Reply:
`new Date(123.123)` can handle decimals. Do we want to still consider it incorrect input?